### PR TITLE
removal of a problematic rule

### PR DESCRIPTION
**Pull request by wout-codes**
https://github.com/AdguardTeam/AdguardFilters/issues/75223#issuecomment-781965506

### CODE REVIEW

**Comment by wout-codes** [comment]
@zloyden Please take a look. It causes breakage.

File: EnglishFilter/sections/adservers.txt [hunk 1]
```diff
@@ -9,7 +9,6 @@
 ||crevicedepressingpumpkin.com^
 ||peuzubuqo.com^$third-party
 ||viqirue.ru^
-||imuled.work^$third-party
```
Comment (zloyden) [inline EnglishFilter/sections/adservers.txt:12]: Unfortunately, not reproduced on my end but there is a good answer https://github.com/AdguardTeam/AdguardFilters/issues/75223#issuecomment-781965506

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/adservers.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:12]: Reproduced it.

Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:12]: How am I supposed two write a scriptlet?

Comment (zloyden) [reply EnglishFilter/sections/adservers.txt:12]: If you meant popups, @AdamWr fixed it here https://github.com/AdguardTeam/AdguardFilters/commit/af4003af928d82b59a81fc4b3940ffc923a081c5

Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:12]: Okay thanks to him.

**Pull request merged**